### DISCUSSION TITLE
Microweber vs Vvveb

### DISCUSSION THREAD

**Discussion by theemack** [givanz/Vvveb#173] [General]
I simply wanted to share my experience: despite Microweber's attractive appearance and feature set, I found its sluggish drag of menu clicks unsatisfactory after trying it out several times for customers. Compared to Vvveb, it's a tortoise. Its slow refresh rates and near 200MB download size have forced me to give up on its attractiveness in favor of Vvveb. Vvveb combines strong features and increased efficiency with a sensible approach to design and simplicity, making it a joy to use for effective design processes.

**Comment by givanz** [comment]
Thank you for review.
